ibm_cloud_sdk_core/base_service.py

```diff
@@ -22,284 +22,100 @@
 import requests
 from requests.structures import CaseInsensitiveDict
 from .version import __version__
-from .utils import has_bad_first_or_last_char, remove_null_values, cleanup_values
-from .iam_token_manager import IAMTokenManager
-from .icp4d_token_manager import ICP4DTokenManager
+from .utils import has_bad_first_or_last_char, remove_null_values, cleanup_values, read_from_external_sources
 from .detailed_response import DetailedResponse
 from .api_exception import ApiException
-
-try:
-    from http.cookiejar import CookieJar  # Python 3
-except ImportError:
-    from cookielib import CookieJar  # Python 2
+from .authenticators import Authenticator, BasicAuthenticator, BearerTokenAuthenticator, CloudPakForDataAuthenticator, IamAuthenticator, NoauthAuthenticator
```

Comment:
Why are you importing all of the authenticators here? I'm not sure you need them